### DISCUSSION TITLE
IMPROVEMENTS THAT CAN CHANGE THE GAME FOR QFIELD

### DISCUSSION THREAD

**Discussion by Sekou225** [opengisch/QField#4649] [Ideas]
Here is a list of some improvement or bugs fixes that i'm sure, can change the game for QField. It's build for field data collection but in my company we use it in a more larger scale.

The main raison we use it is to FOLLOW our drilling program progress : the dayly changes are entered by the geologist while the office workers on site and HQ can see them on their app. **_AUTOMATIC SYNCHRONIZATION WILL BE VERY HELPFULL IN THIS FIELD_** 

The geology and construction workers can use it to reach a certain point on the field ( particilar drill locations). **_ADDING GEOMETRY (points, nodes) to a layer in edition mode_** **_BY ENTERING COORDINATES WILL ALSO BE AMAZING_**

The dayly, weekly and monthly reports are enriched by maps exported from Qfield ( different map layout and altlas layout to fit everyone's needs) in this point, _**IT WOULD BE HELPFULL TO LET PEOPLE CHOOSE THEIR EXPORT FORMAT (PDF OR JPG OR PNG)**_

**_Other thing is the capability to Filter faetures by attributes or expressions._**

**Comment by Spacedatists** [comment]
> The main raison we use it is to FOLLOW our drilling program progress : the dayly changes are entered by the geologist while the office workers on site and HQ can see them on their app. **_AUTOMATIC SYNCHRONIZATION WILL BE VERY HELPFULL IN THIS FIELD_**

See my [request ](https://ideas.qfield.org/app-feature-requests/p/change-directory-of-imported-projects-folder-on-android-devices/voters)here, since we have been using QFIELD for data collection on a large scale (100+ devices) and urgently need automated Syncs back!

**Reply by Sekou225** [reply]
May you could share an other link. This one looks bad and point on no expected page.

**Comment by ZsanettMed** [comment]
Hello @Sekou225!

We really appreciate you sharing your thoughts on QField. It's great to get new ideas from users. Good news: as of version 3.2 Congo, cloud projects have the ability to automatically sync changes regularly.

For any additional suggestions, feel free to post them or upvote existing ones on our [QField Idea Platform](https://ideas.qfield.org/). 

If you're interested in speed up the development of a feature, feel free to drop us a mail to sales@qfield.org. 

PS. I'm closing this discussion here since we are moving all requests to the QField Idea Platform. :)

Thanks and best wishes,

Zsanett
